Description: Adapt documentation build process for Debian packaging purposes
Author: Sandro Tosi <morph@debian.org>
Origin: vendor
Forwarded: not-needed
Last-Update: 2014-02-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -142,6 +142,7 @@
 	PYTHONSUFFIX=$$(python -V 2>&1 |awk '{print $$2}' |awk -F. '{print $$1 "." $$2}') && \
 	PYTHONPATH=$$(ls -d build/lib.*$$PYTHONSUFFIX):$$PYTHONPATH \
 	sphinx-build doc build/doc
+	rst2html README.rst build/doc/README.html
 
 # Rebuild all documentation.
 # As sphinx extracts documentation from pycurl modules, docs targets
